### PR TITLE
Release toolchain configs for Bazel 0.15.0.

### PR DESCRIPTION
**Pull request by xingao267**

### CODE REVIEW

File: bazelrc/bazel-0.15.0.bazelrc [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This .bazelrc file contains all of the flags required for the toolchain,
+# Remote Build Execution, and the Bazel Build Results UI. Specific flags in
+# your Bazel command allow you to use only the remote build, to use only the
+# results UI, or to use them both together.
```
[the remark this answers is not in the document]
Comment (xingao267) [reply bazelrc/bazel-0.15.0.bazelrc]: Done.

File: bazelrc/bazel-0.15.0.bazelrc [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+# and need to be adapted to work with other toolchain containers.
+build:remote --host_javabase=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:jdk8
+build:remote --javabase=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:jdk8
+build:remote --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --crosstool_top=@bazel_toolchains//configs/ubuntu16_04_clang/1.0/bazel_0.15.0/default:toolchain
+build:remote --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
+build:remote --enabled_toolchain_types=@bazel_tools//tools/cpp:toolchain_type
+build:remote --extra_toolchains=@bazel_toolchains//configs/ubuntu16_04_clang/1.0/bazel_0.15.0/cpp:cc-toolchain-clang-x86_64-default
+build:remote --extra_execution_platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:rbe_ubuntu1604
```
[the remark this answers is not in the document]
Comment (xingao267) [reply bazelrc/bazel-0.15.0.bazelrc]: Done.

File: configs/debian8_clang/0.3.0/toolchain.bazelrc [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 build:remote --host_javabase=@bazel_toolchains//configs/debian8_clang/0.3.0:jdk8
 build:remote --javabase=@bazel_toolchains//configs/debian8_clang/0.3.0:jdk8
-build:remote --crosstool_top=@bazel_toolchains//configs/debian8_clang/0.3.0/bazel_0.14.1/default:toolchain
-build:remote --experimental_remote_platform_override='properties:{ name:"container-image" value:"docker://gcr.io/cloud-marketplace/google/rbe-debian8@sha256:0d5db936f8fa04638ca31e4fc117415068dca43dc343d605c0db2a15f433a327" }'
+build:remote --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --crosstool_top=@bazel_toolchains//configs/debian8_clang/0.3.0/bazel_0.15.0/default:toolchain
 build:remote --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
-build:remote --extra_toolchains=@bazel_toolchains//configs/debian8_clang/0.3.0/bazel_0.14.1/cpp:cc-toolchain-clang-x86_64-default
+build:remote --enabled_toolchain_types=@bazel_tools//tools/cpp:toolchain_type
```
[the remark this answers is not in the document]
Comment (xingao267) [reply configs/debian8_clang/0.3.0/toolchain.bazelrc]: Done

File: configs/debian8_clang/0.3.0/toolchain.bazelrc [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
+build:remote --extra_toolchains=@bazel_toolchains//configs/debian8_clang/0.3.0/bazel_0.15.0/cpp:cc-toolchain-clang-x86_64-default
 build:remote --extra_execution_platforms=@bazel_toolchains//configs/debian8_clang/0.3.0:rbe_debian8
+build:remote --host_platform=@bazel_toolchains//configs/debian8_clang/0.3.0:rbe_debian8
+build:remote --platforms=@bazel_toolchains//configs/debian8_clang/0.3.0:rbe_debian8
 
 # Experimental configs for sanitizers, use --config=remote,remote-xxsan,remote-<asan/tsan/msan> (in that order)
 build:remote-xxsan --copt=-gmlt
```
[the remark this answers is not in the document]
Comment (xingao267) [reply configs/debian8_clang/0.3.0/toolchain.bazelrc]: Done.

File: configs/ubuntu16_04_clang/1.0/toolchain.bazelrc [hunk 5]
```diff
@@ -1,11 +1,15 @@
...
 build:remote --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
-build:remote --extra_toolchains=@bazel_toolchains//configs/ubuntu16_04_clang/1.0/bazel_0.14.1/cpp:cc-toolchain-clang-x86_64-default
+build:remote --enabled_toolchain_types=@bazel_tools//tools/cpp:toolchain_type
+build:remote --extra_toolchains=@bazel_toolchains//configs/ubuntu16_04_clang/1.0/bazel_0.15.0/cpp:cc-toolchain-clang-x86_64-default
 build:remote --extra_execution_platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:rbe_ubuntu1604
+build:remote --host_platform=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:rbe_ubuntu1604
+build:remote --platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:rbe_ubuntu1604
 
 # Experimental configs for sanitizers, use --config=remote,remote-xxsan,remote-<asan/tsan/msan> (in that order)
 build:remote-xxsan --copt=-gmlt
```
[the remark this answers is not in the document]
Comment (xingao267) [reply configs/ubuntu16_04_clang/1.0/toolchain.bazelrc]: Done.

File: configs/ubuntu16_04_clang/1.0/toolchain.bazelrc [hunk 6]
```diff
@@ -1,11 +1,15 @@
 # Toolchain related flags to append at the end of your .bazelrc file.
 build:remote --host_javabase=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:jdk8
 build:remote --javabase=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:jdk8
-build:remote --crosstool_top=@bazel_toolchains//configs/ubuntu16_04_clang/1.0/bazel_0.14.1/default:toolchain
-build:remote --experimental_remote_platform_override='properties:{ name:"container-image" value:"docker://gcr.io/cloud-marketplace/google/rbe-ubuntu16-04@sha256:59bf0e191a6b5cc1ab62c2224c810681d1326bad5a27b1d36c9f40113e79da7f" }'
+build:remote --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --crosstool_top=@bazel_toolchains//configs/ubuntu16_04_clang/1.0/bazel_0.15.0/default:toolchain
 build:remote --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
-build:remote --extra_toolchains=@bazel_toolchains//configs/ubuntu16_04_clang/1.0/bazel_0.14.1/cpp:cc-toolchain-clang-x86_64-default
```
[the remark this answers is not in the document]
Comment (xingao267) [reply configs/ubuntu16_04_clang/1.0/toolchain.bazelrc]: Done.

**Comment by nlopezgi** [comment]
thanks for checking @tmarij4

**Pull request merged**